lib/dashboard-template.hbs

```diff
@@ -1009,7 +1014,9 @@ body.theme-light code {
                 {{/with}}
                 {{/isNotIn}}
                 {{/unless}}
+                {{/unless}}}
```

Comment:
There's an extra `}` on the end and that would cause a failure when run

Reply:
Oh, I oversaw that. It's fixed.